config/authentication/group.cfg

```diff
@@ -1,18 +1,19 @@
+# Assignations required for a working installation
 admin:admin
-demo:admin
+node:node
 rm:admin
-
-provider:providers
-
-guest:guests
-user:user
 scheduler:user
+watcher:watchers
 
-radmin:rmcoreadmins
-nsadmin:nsadmins
+# The following group assignations are there for testing purposes
+# Please comment, remove or add assignations based on your needs below
 admin:nsadmins
+demo:admin
+guest:guests
+nsadmin:nsadmins
+provider:providers
+radmin:rmcoreadmins
 rm:nsadmins
-
-watcher:watchers
-
 test_executor:admin
+user:user
```

Comment:
I'm not sure that user "user" is only for testing purpose, it is also meant to be an account which has standard user privilege.